### ISSUE TITLE
Add tests for retrieving Items from collections

### ISSUE THREAD

**Issue by vddesai1871** [HTTP-APIs/hydrus#377]
### I'm submitting a 
- [x] bug report.
- [x] feature request.

### Current Behaviour:

~We have tests for checking operations over individual members of collection, but we do not have any test to check response for collection (for example checking response for api/XyzCollection).~
Add test to check `member` field of collection response holds valid items/elements. 
### Expected Behaviour:

We need a simple test to check status code returned by dereferencing a member element.
### Steps to reproduce:

In crud.py line 559  and 563 sets the uri of member elements. 
```
"@id": "/{}/{}/{}".format(API_NAME, path, instance_.id),
```
change it to anything else like
```
"@id": "/{}/xyz/".format(API_NAME, path, instance_.id),
````
every tests will pass as it is not being tested in any of the tests.
### Do you want to work on this issue?

yes

**Comment by chrizandr** [comment]
We do have `test_Collections_GET` and `test_Collections_PUT` in `test_app.py`.

Is there anything I am missing? I added those tests a long time ago in #48 . We are already checking responses for every collection, what seems to be missing there?

**Comment by vddesai1871** [comment]
Yes the test I mentioned in the issue is already in test_Collection_GET, I do not remember how I missed it. Still I'll reevaluate it and then close it if I find nothing.

**Comment by vddesai1871** [comment]
I think it was for checking the validity of member item URIs in the collection response. I encountered it while working on #378 , but yes the description I gave was wrong, I have updated it and I am creating a PR for it.

**Comment by chrizandr** [comment]
Merged in #398
